### DISCUSSION TITLE
Is it possible to run ISCE2 - StripMapStack without Multilooking i.e. -range_looks 1 -azimuth_looks 1 ?

### DISCUSSION THREAD

**Discussion by tonianr** [isce-framework/isce2#394] [Q&A]
I'm attempting to stack TSX images using StripmapStack with no multilooking:

!stackStripMap.py --slc_directory /home/toni/WCF_TerraSARX_processing/SRTM_PS_processing/HS_029/SLC/ \
    --dem /home/toni/WCF_TerraSARX_processing/SRTM_PS_processing/HS_029/DEM/demLat*.dem.wgs84 \
    --azimuth_looks 1 \
    --range_looks 1 \
    --time_threshold 200 \
    --baseline_threshold 200 \
    --workflow slc \
    --unw_method no \
    --nofocus \
    --zero

I however keep getting the error after running **! run_files/run_01_reference**: 
_**FileNotFoundError: [Errno 2] No such file or directory: '/home/toni/WCF_TerraSARX_processing/SRTM_PS_processing/HS_029/geom_reference/lat.rdr.xml'**_

The **run_01_reference** does not create a _**/geom_reference**_ directory in my main dir _HS_029_, but there is one in my _**merged/geom_reference**_ directory.

How would it affect my processing if I copied the geom_reference from my merged dir to my main directory and re-ran run_01_reference?

**Comment by EJFielding** [comment]
I don't know the details of what you need to do to process without taking looks, but I think the FRINGE software must be doing this:
 https://github.com/isce-framework/fringe

**Comment by volter29** [comment]
I EJFielding also have such a request.
For research reasons I should do an analysis on an archaeological site to monitor even "small" structures. So I prefer do to process without taking look. is it possible not to use Multilooking for StripmapStack? And then use STAMPS?

**Reply by tonianr** [reply]
This is also my dilemma! I continued with 2x2 multi looking for now.

**Comment by nadiahassanpour** [comment]
> I'm attempting to stack TSX images using StripmapStack with no multi looking:
> 
> !stackStripMap.py --slc_directory /home/toni/WCF_TerraSARX_processing/SRTM_PS_processing/HS_029/SLC/ --dem /home/toni/WCF_TerraSARX_processing/SRTM_PS_processing/HS_029/DEM/demLat*.dem.wgs84 --azimuth_looks 1 --range_looks 1 --time_threshold 200 --baseline_threshold 200 --workflow slc --unw_method no --nofocus --zero
> 
> I however keep getting the error after running **! run_files/run_01_reference**: _**FileNotFoundError: [Errno 2] No such file or directory: '/home/toni/WCF_TerraSARX_processing/SRTM_PS_processing/HS_029/geom_reference/lat.rdr.xml'**_
> 
> The **run_01_reference** does not create a _**/geom_reference**_ directory in my main dir _HS_029_, but there is one in my _**merged/geom_reference**_ directory.
> 
> How would it affect my processing if I copied the geom_reference from my merged dir to my main directory and re-ran run_01_reference?

Hi Tonianr, I wanted to run my TSX data without multi-looking, but I got the error like you. Did you get your answer? could you run your isce 1*1?

**Reply by tonianr** [reply]
I used a 2x2 multi-looking instead!

**Reply by EJFielding** [reply]
Yes, I believe that the ISCE2 stripmapStack workflow will only work if you do some multilooking. You would need to use other software if you want to complete the InSAR analysis at the full SLC 1-look resolution, such as the FRINGE software.

**Reply by nadiahassanpour** [reply]
thanks all. I used a 2*2 multi-looking.
